Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -45,6 +45,13 @@
                          *globalValidation
                          *hltvalidation)
 
+# fastsim: remove bits that depend on digis, tracker simhits or tracker rechits
+def _validation_fastSimMods(sequence):
+    sequence.remove(globaldigisanalyze)
+    sequence.remove(globalhitsanalyze)
+    sequence.remove(globalrechitsanalyze)
+eras.fastSim.toModify(validation,func=_validation_fastSimMods)
```

Comment:
@Dr15Jones I have also a need to remove multiple elements of Sequence in some further work for `TrackValidation_cff.py`. In my private branch I did this
```
someSequence = cms.Sequence(a+b+c+d)
def _removeMany(seq, lst):
    for item in lst:
        seq.remove(item)
eras.fastSim.toModify(someSequence, lambda x: _removeMany(x, [b, d]))
```
but given also that Lukas did this three times in this PR, would it make sense to provide a generic way/pattern to remove multiple elements in one `toModify()` call?